src/components/BackButton.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+    <div class="back-button" :style="`left:${x}; top:${y};`" v-on:click="nav">
```

Comment:
Looks like Andi started out indenting templates 4 spaces. Let's stay consistent with that.